docs/components/example.css

```diff
@@ -48,6 +48,10 @@ body.cauldron--theme-dark {
   margin: 0;
 }
 
+.Component__example__react .Notice {
+  line-height: 1.618;
+}
```

Comment:
Why `line-height`, shouldn't we be rendering the component as is?

Reply:
It's specifically an issue when a string gets passed in without an HTML element, e.g., 

```ts

<Notice type='info' title='Information'>A bunch of text that doesn't have a parent element such as p, span, etc.</Notice>

```

The line height for the text is super condensed - much more so than the line height we apply to paragraph elements - and really tough to target with styles outside of doing something potentially too generic that effects elements we don't want it to. 

I could try to hide it by just keeping the content blocks short instead of lengths of text that would wrap, then let the end-user determine the appropriate solution. 

I went with adding the style here as this is a problem now run into twice (with the Panel as well) and not having a reset for margins (e.g., the ones in Chrome for paragraphs) or line height caused similar issues. Seemed a low impact way to solve it, without strictly requiring those styles in the component itself.

Reply:
This seems like an implementation issue and shouldn't be something that's just fixed in documentation. I would expect the component structure that you passed in to work out of the box without any modifications on the developer side.

Reply:
I'm fine with that, but if that's the case, I recommend adding more universal line-height for text containing elements than just `<p>`. This is not just an issue here, it is an issue with most other components as well, the docs just don't include long enough copy for it to be really seen (case in point, the `Panel` example copy is one word, the `ExpandCollapse` and `Accordion` have similar issues. 

[Here is where](https://github.com/dequelabs/cauldron/blob/10f53c90fe2466f913bb00c49fc0b8e1aad501b1/packages/styles/base.css#L84-L86) the `line-height` for the `<p>` comes from (although you probably already know that...)

Here's a comparison:

## Text content
<img width="1000" alt="CleanShot 2023-08-01 at 16 27 42@2x" src="https://github.com/dequelabs/cauldron/assets/1016306/39d5722c-7d8b-4bda-929b-8ebb98284a6d">

## Paragraph
<img width="983" alt="CleanShot 2023-08-01 at 16 36 06@2x" src="https://github.com/dequelabs/cauldron/assets/1016306/b0cc81ca-5f02-4ea2-b61e-c2179fc0e510">